crates/drainer/src/lib.rs

```
@@ -55,9 +58,17 @@ async fn drainer(
 
     // parse_stream_entries returns error if no entries is found, handle it
     let (entries, last_entry_id) = utils::parse_stream_entries(&stream_read, stream_name)?;
-
     let read_count = entries.len();
 
+    metrics::JOBS_PICKED_PER_STREAM.add(
+        &metrics::CONTEXT,
+        u64::try_from(read_count).unwrap_or(u64::MAX),
```

Preferably use `u64::MIN` here so that we can ignore errors